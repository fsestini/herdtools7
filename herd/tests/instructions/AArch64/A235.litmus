AArch64 A235
Variant=morello

{
int x=1;
0:X1=x;
}

  P0           ;
  STR WZR,[C1] ;
forall [x]=0
