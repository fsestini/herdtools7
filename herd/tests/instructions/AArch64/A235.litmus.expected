Test A235 Required
States 1
[x]=1;
No
Witnesses
Positive: 0 Negative: 1
Condition forall ([x]=0)
Observation A235 Never 0 1
Hash=63d39367a4fe571201587275c10c0941

